index.html

```diff
@@ -289,6 +289,28 @@ <h2>About me</h2>
           </ul>
         </div>
       </div>
+      <footer class="contact_form">
+        <h2 class="contact_header">
+          I'm always interested in hearing
+           about new projects, 
+          so if you'd like to
+           chat please get in touch.
+        </h2>
+        <form action="#" method="post" class="form">
```

Comment:
- [ ] As the previous reviewer said, Please let's collect data using `formspree` :smile:.  Formspree is a service that helps us to collect data from the user.[ you can read more about this service.](https://formspree.io/)